rfb_file.js

```diff
@@ -263,21 +263,38 @@ export function createBalanceReportData(obj) {
         document,
         fullname,
         address,
-    
+
         fiat_balance,
-    
-        coin_symbol,
-        coin_balance,
+
+        coin_balances,
+
+        coin_symbol: old_coin_symbol,
+        coin_balance: old_coin_balance,
     } = obj;
 
-    const rfb_fiat_balance = fiat_balance.toFixed(2).replace(/\./g, '');
+    if (!coin_balances) coin_balances = [];
+
+    if (coin_symbol && coin_balance) {
+        coin_balances.push({
+            coin_symbol: old_coin_symbol,
+            coin_balance: old_coin_balance,
+        });
+    }
+
+    let returnString = `${line_type_1}|${formatDate(date)}|${rfb_identity_type}|${country}|${rfb_cpf}|${rfb_nif}|${fullname}|${address}|${rfb_fiat_balance}\r\n`;
 
-    const rfb_coin_balance = coin_balance.toFixed(10).replace(/\./g, '');
+    for ({ coin_symbol, coin_balance } of coin_balances) {
+        const rfb_coin_balance = coin_balance.toFixed(10).replace(/\./g, '');
+
+        returnString += `${line_type_2}|${formatDate(date)}|${rfb_identity_type}|${country}|${rfb_cpf}|${rfb_nif}|${coin_symbol}|${rfb_coin_balance}\r\n`;
```

Comment:
This line requires one `rfb_cpf` and `rfb_nif` and `rfb_identity_type`. These are defined from line 294 and beyond. Is this ok?

Reply:
fixed